tools/Customization/DevHome.Customization/Views/OptimizeDevDriveDialog.xaml

```diff
@@ -0,0 +1,29 @@
+<!-- Copyright (c) Microsoft Corporation.. -->
+<!-- Licensed under the MIT License. -->
+
+<ContentDialog
+    x:Class="DevHome.Customization.Views.OptimizeDevDriveDialog"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+    xmlns:environments="using:DevHome.Customization.ViewModels.Environments" 
+    xmlns:i="using:Microsoft.Xaml.Interactivity"
+    xmlns:ic="using:Microsoft.Xaml.Interactions.Core"
+    d:DataContext="{d:DesignInstance Type=environments:DevDriveOptimizerCardViewModel}"
+    x:Uid="ms-resource:///DevHome.Customization/Views/OptimizeDevDriveDialog"
+    mc:Ignorable="d"
+    Style="{StaticResource DefaultContentDialogStyle}"
+    Title="{x:Bind ViewModel.ChooseDirectoryPromptText}"
+    PrimaryButtonText="Make changes"
+    PrimaryButtonCommand="{x:Bind ViewModel.DirectoryInputConfirmedCommand}"
+    SecondaryButtonText="Cancel">
+    <StackPanel>
+        <TextBlock Text="{x:Bind ViewModel.OptimizeDevDriveDialogDescription}" TextWrapping="Wrap"/>
+        <Grid Margin="0 10 0 0">
+            <TextBox HorizontalAlignment="Left" Width="300" Grid.Column="0" x:Name="DirectoryPathTextBox" PlaceholderText="Choose a directory" Text="{x:Bind ViewModel.DirectoryPathTextBox , Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"/>

```

Comment:
Suggest using MinWidth, so that it can scale up in size if text scale is turned up (preventing accessibility bug)